app/assets/stylesheets/custom.scss

```diff
@@ -22,6 +22,29 @@ main {
   padding: 20px;
 }
 
+.card .card-image img {
+  position: center;
+  width: 30%;
+}
+
+.card-content h3 {
+  font-size: 25px;
+  margin-bottom: 0;
+  color: #303F9F;
+  font-family: 'Montserrat', sans-serif;
+  }
+  
+.card-content h4 {
```

Comment:
@johnedelmapa 
In Sass, you can nest styles to avoid duplication for `.card-content`